src/test/scala/com/ubirch/services/IncidentListenerSpec.scala

```diff
@@ -62,12 +71,12 @@ class IncidentListenerSpec extends TestBase with EmbeddedKafka with StrictLoggin
       }
     }
     val fakeTenantRetriever = new FakeTR(config, mockRedis)
-    var incidentList: Seq[Array[Byte]] = Seq()
+    val incidentList: ListBuffer[Array[Byte]] = ListBuffer.empty[Array[Byte]]
```

Comment:
I changed it because there was warning.
```
[WARNING] /Users/hirokifujino/Documents/ubirch/repos/ubirch-incident-service/src/test/scala/com/ubirch/services/IncidentListenerSpec.scala:75: warning: assignments in argument position are deprecated in favor of named arguments. Wrap the assignment in brackets, e.g., `{ incidentList = ... }`.
[WARNING]         synchronized(incidentList = incidentList :+ incident)
[WARNING]    
```

Reply:
Ok, good. Would be interesting to understand the issue.

Reply:
Ah, now I understand.